tests/pytests/test_mdis_drivers.py

```diff
@@ -20,57 +20,165 @@ def test_kernels():
+            'InstrumentPosition': {
+                'SpkTableStartTime': 483122606.85252,
+                'SpkTableEndTime': 483122606.85252,
+                'SpkTableOriginalSize': 1,
+                'EphemerisTimes': [483122606.85252],
+                'Positions': [[1844.0372753409,-966.47836059939,1322.8442926008]],
+                'Velocities': [[-2.6146204621248,-0.30506719264668,2.8563617189104]]},
+            'SunPosition': {
+                'SpkTableStartTime': 483122606.85252,
+                'SpkTableEndTime': 483122606.85252,
+                'SpkTableOriginalSize': 1,
+                'EphemerisTimes': [483122606.85252],
+                'Positions': [[11805115.780304,-39513889.220831,-22331586.824705]],
+                'Velocities': [[56.903276907758496, 11.401211521119633, 0.19123815749182813]]}},
+        'usgscsm': {
+            'radii': {
+                'semimajor': 2439.4,
+                'semiminor': 2439.4,
+                'unit': 'km'},
+            'sensor_position': {
+                'positions': np.array([[-629496.4862139452, -1582946.0091313303, 1783952.6403104223]]),
+                'velocities': np.array([[1732.186663134826, 2502.768253236102, 2412.6523222267365]]),
+                'unit': 'm'},
+            'sun_position': {
+                'positions': np.array([[-4.68946673e+10, -5.36158427e+08,  2.71167863e+07]]),
+                'velocities': np.array([[-4629.73346128, 256.72086237, 10.63960444]]),
+                'unit': 'm'},
+            'sensor_orientation': {
+                'quaternions': np.array([[-0.9341837162001089, -0.0014477307579469636, 0.0044938223368195815, 0.3567611161870253]])},
+            'detector_sample_summing': 2,
+            'detector_line_summing': 2,
+            'focal_length_model': {
+                'focal_length': 549.5535053027719},
+            'detector_center': {
+                'line': 512,
+                'sample': 512},
+            'starting_detector_line': 1,
+            'starting_detector_sample': 9,
+            'focal2pixel_lines': [0.0, 0.0, 71.42857143],
+            'focal2pixel_samples': [0.0, 71.42857143, 0.0],
+            'optical_distortion': {
+                'transverse': {
+                    'x': [0.0, 1.001854269623802, 0.0, 0.0, -0.0005094440474941111, 0.0, 1.004010471468856e-05, 0.0, 1.004010471468856e-05, 0.0],
+                    'y': [0.0, 0.0, 1.0, 0.0009060010594996751, 0.0, 0.0003574842626620758, 0.0, 1.004010471468856e-05, 0.0, 1.004010471468856e-05]}},
+            'image_lines': 512,
+            'image_samples': 512,
+            'name_platform': 'MESSENGER',
+            'name_sensor': 'MERCURY DUAL IMAGING SYSTEM NARROW ANGLE CAMERA',
+            'reference_height': {
+                'maxheight': 1000,
+                'minheight': -1000,
+                'unit': 'm'},
+            'name_model': 'USGS_ASTRO_FRAME_SENSOR_MODEL',
+            'center_ephemeris_time': 483122606.85252464}}}
```

Comment:
Do people like using a big dictionary like this to parameterize the loads test? I'd like to add an MDIS WAC image to the test.

Reply:
To get the dict, are you just dumping an existing one and pasting it into the test?

Reply:
The new ISIS dict was generated by running tabledump on the spiceinit'd test image. The only changes are flipping signs on quaternions and adding extra values in the NaifKeywords.

The pre-existing USGSCSM dict is just the old comparison dictionary from before this PR.

Reply:
I think doing it this way is fine and straightforward. We're going to have large dictionaries regardless.